extension/public/worker/twitter.js

```diff
@@ -0,0 +1,14 @@
+/** @param {HTMLDivElement} el */
+function checkTweet(el) {
+  if (window.matchesBlocklist(el.innerText)) {
+    // eslint-disable-next-line no-param-reassign
```

Comment:
What does this comment do?

Reply:
sorry, I've removed this comment. It silences a lint warnings from eslint, a bit like `@SuppressWarnings` in java. I've changed the config to disable this warning altogether, and I think we could even remove eslint once sonarlint/sonarcloud is properly configured (#75)